lib/cnet/tcp/tcp_input.c

```diff
@@ -90,14 +92,14 @@ tcp_input_lookup(struct cne_node *node, pktmbuf_t *m, struct pcb_hd *hd)
     pcb = cnet_pcb_lookup(hd, &key, BEST_MATCH);
     if (likely(pcb)) {
         int rc = TCP_INPUT_NEXT_PKT_DROP;
-
+#if 0        // Disable till cne_ipv4_udptcp_cksum_verify is fixed
         if (is_pcb_dom_inet6(pcb))
             csum = cne_ipv6_udptcp_cksum_verify(&tip->ip6, &tip->tcp);
         else
             csum = cne_ipv4_udptcp_cksum_verify(&tip->ip4, &tip->tcp);
         if (csum)
```

Comment:
Also csum here needs to be signed, at the fns above return -1 on a fail...

Reply:
Yes I agree it should be signed, but maybe the function should be returning a 0 or 1 for pass/fail?

Reply:
maybe I should just rename the `csum` variable to something else as it's not really a csum returned by those functions

Reply:
Yes, I agree it is not a checksum. We can do that change now or after this one gets merged up to you or I can create a PR for that just that fix.

Reply:
I can push a separate PR to rename it after this no prob